util/Makefrag

```diff
@@ -56,6 +86,22 @@ work/lib/libfesvr.a: work/${FESVR_VERSION}_unzip
 	mkdir -p $(dir $@)
 	cp $(dir $<)libfesvr.a $@
 
+# Build fesvr seperately for verilator since this might use different compilers
+# and libraries than modelsim/vcs and
+$(VLT_FESVR)/${FESVR_VERSION}_unzip:
+	mkdir -p $(dir $@)
+	wget -O $(dir $@)/${FESVR_VERSION} https://github.com/riscv/riscv-isa-sim/tarball/${FESVR_VERSION}
+	tar xfm $(dir $@)${FESVR_VERSION} --strip-components=1 -C $(dir $@)
+	patch -d $(dir $@) -p1 < ${ROOT}/util/patches/riscv-isa-sim/fesrv.patch
```

Comment:
Does this also work with gcc or is this something that should only happen with LLVM?

Reply:
Ah sorry, I was thinking about the patch you apply. But if the GCC flow still works with the patch then I am more than happy :)

Reply:
This should clarify:

|      | gcc | llvm |
|------|-----|------|
| vsim | `ucontext` | not implemented |
| vlt  | `pthread` | `pthread` |

Reply:
Ah perfect. Thanks :)